File: 프로그래머스/0/120883. 로그인 성공？/로그인 성공？.js
function solution(id_pw, db) {  
    return db.some((x) => x[0]===id_pw[0]) ? (db.some((x) => x[0]===id_pw[0] && x[1]===id_pw[1]) ? "login" : "wrong pw") : "fail";
}

File: 프로그래머스/0/120883. 로그인 성공？/로그인 성공？.py
def solution(id_pw, db):
    id, pw = id_pw[0], id_pw[1]
    answer = 0

    for [user_id, user_pw] in db:
        if user_id == id and user_pw == pw:
            return "login"
        elif user_id == id and user_pw != pw:
            answer = 1

    return "wrong pw" if answer == 1 else "fail"